COSC481W-2024Fall/MLite
# Added rails code for dummy model recommendations

Features:
- Add model recommendation system to recommend based on dataset metadata
- Added logical flow for `models/new`: select dataset > select target column > select one of the recommended models

Testing:
- Go to `models/new`
- Select a dataset
- Select a target column
- You should see a few suggested models and hyperparams
- Select a model, fill in name and description, and create the model
- You should now be in the `model#show` page, with info on model_type, hyperparams, and labels

Issue #26
